demo/App.tsx

```diff
@@ -6,10 +6,82 @@
  *
  */
 
+import * as Constants from './src/constants';
+import Hyperview from 'hyperview';
 import './src/gesture-handler';
+import React, { useState } from 'react';
+import { fetchWrapper, formatDate } from './src/helpers';
+import {
+  SafeAreaInsetsContext,
+  SafeAreaProvider,
+} from 'react-native-safe-area-context';
+import HyperviewScreen from './src/HyperviewScreen';
+import { View } from 'react-native';
 import Navigator from './src/Navigator';
-import React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
 
-export default function App() {
-  return <Navigator />;
+const NAMESPACE_URI = 'https://instawork.com/hyperview-set-navigator-source';
+
+/**
+ * Define expected values
+ */
+enum NavigationSource {
+  HXML = 'hxml',
+  EXTERNAL = 'external',
```

Comment:
Maybe instead of `html`, `hyperview` would be more representative. Also sorting alphabetically:
```suggestion
  EXTERNAL = 'external',
  HYPERVIEW = 'hyperview',
```